CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Possible log types:
 - `[fixed]` for any bug fixes.
 - `[security]` to invite users to upgrade in case of vulnerabilities.
 
+## Unreleased
+
+ - [changed] Type annotations: Change parameter order
+   from `Result[E, T]` to `Result[T, E]` to match Rust/OCaml/F# (#7)
```

Comment:
```suggestion
   from `Result[E, T]` to `Result[T, E]` to match Rust/OCaml/F# (#7)
 - [added] Add `map`, `map_or`, `map_or_else`, and `map_err` (#19)
```

Forgot to update changelog during #19, we can either sneak it into this or if you prefer, I can update changelog after this is merged and before 0.5.0 is released.

Reply:
I usually update the CHANGELOG just before releasing, so no worries. In this case I already added the CHANGELOG entry mainly because the RP contains a breaking change, to make absolutely sure that we don't forget about it when releasing :slightly_smiling_face: